strr-base-web/app/error.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import type { NuxtError } from '#app'
+
+const { t } = useI18n()
+const localePath = useLocalePath()
+
+const props = defineProps({
+  error: { type: Object as () => NuxtError, default: undefined }
+})
+
+// TODO: update with other error codes?
+const errorKey: string | number = props.error?.statusCode === 404 ? 404 : 'unknown'
+
+// cant use definePageMeta in error.vue
+useRoute().meta.hideBreadcrumbs = true
+
+useHead({
+  title: errorKey === 404 ? t('page.error.404.title') : t('page.error.unknown.title')
+})
+
+</script>
+
+<template>
+  <NuxtLayout>
+    <div class="m-auto flex flex-col items-center gap-4">
+      <h1>
+        {{ $t(`page.error.${errorKey}.h1`) }}
+      </h1>
+      <p>{{ $t(`page.error.${errorKey}.content`) }}</p>
```

Comment:
Is there a default for strr?

Reply:
Not sure what you mean by this.

Reply:
This is for 404 pages

Defaults to 'unknown',

```
error: {
      404: {
        title: 'Page Not Found - Short-Term Rental Registry',
        h1: '404 Page Not Found',
        content: 'This page could not be found or does not exist.'
      },
      unknown: {
        title: 'Unknown Error - Short-Term Rental Registry',
        h1: 'Unknown Error',
        content: 'An unknown error occured, please refresh the page or try again later.'
      }
    },
```